conf/machine/include/tegra-common.inc

```diff
@@ -34,6 +34,9 @@ PREFERRED_PROVIDER_v4l-utils = "${@'v4l-utils' if 'openembedded-layer' in d.getV
 PREFERRED_PROVIDER_tensorrt-plugins ?= "tensorrt-plugins-prebuilt"
 PREFERRED_PROVIDER_tensorrt-trtexec ?= "tensorrt-trtexec-prebuilt"
 PREFERRED_RPROVIDER_tegra-gbm-backend ?= "tegra-libraries-gbm-backend"
+PREFERRED_RPROVIDER_optee-client-tegra ?= "optee-client-tegra"
+PREFERRED_RPROVIDER_optee-samples ?= "optee-samples"
+PREFERRED_RPROVIDER_optee-test-tegra ?= "optee-test-tegra"
```

Comment:
See above.  Stick with prebuilt as default for now, until the issues with the from-source build are straightened out.

Also, these, should be PREFERRED_PROVIDER, not PREFERRED_RPROVIDER, at least for the client and sample recipes.  I'm seeing this during my builds:
```
NOTE: Resolving any missing task queue dependencies
NOTE: Multiple providers are available for optee-samples (optee-samples, optee-samples-prebuilt)
Consider defining a PREFERRED_PROVIDER entry to match optee-samples
NOTE: Multiple providers are available for optee-client-tegra (optee-client-tegra, optee-client-tegra-prebuilt)
Consider defining a PREFERRED_PROVIDER entry to match optee-client-tegra
```

Reply:
And actually, as I mentioned in the summary comment for the review, the sample shouldn't be using PROVIDES/RPROVIDES anyway.

Reply:
I don't see the same thing when starting fresh build.
I will switch all to `prebuild` for now until we will have a fix for the recipes from source.

Reply:
Sorry, I double check and I confirm the same thing as you 

```
➜  build-orin-agx git:(wip-jetpack-5-ga) ✗ bitbake demo-image-base -f      
Loading cache: 100% |                                                                                                                                                                        | ETA:  --:--:--
Loaded 0 entries from dependency cache.
Parsing recipes: 100% |#######################################################################################################################################################################| Time: 0:00:14
Parsing of 2820 .bb files complete (0 cached, 2820 parsed). 4351 targets, 118 skipped, 0 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies
NOTE: Multiple providers are available for optee-client-tegra (optee-client-tegra, optee-client-tegra-prebuilt)
Consider defining a PREFERRED_PROVIDER entry to match optee-client-tegra
NOTE: Multiple providers are available for optee-samples (optee-samples, optee-samples-prebuilt)
Consider defining a PREFERRED_PROVIDER entry to match optee-samples

```